lib/Db/LocalMessage.php

```diff
@@ -45,6 +45,8 @@
  * @method void setBody(string $body)
  * @method bool isHtml()
  * @method void setHtml(bool $html)
+ * @method bool isFailed()
```

Comment:
```suggestion
 * @method bool|null isFailed()
```

Reply:
thanks oracle

Reply:
we could overwrite fromRow for the entity class and cast the value to bool. That's a bit nicer than our === true checks.

Reply:
If I'm not mistaken then our `Entity::setter` casts the nullable bool correctly: https://3v4l.org/D2AKf. `null` becomes `false` with `settype`.

Reply:
Nevermind. `null` stays `null`.

Reply:
> Nevermind. null stays null.

Been there :disappointed: